main.go

```diff
@@ -62,11 +90,24 @@ func main() {
 		prometheus.Unregister(prometheus.NewGoCollector())
 	}
 
-	fullListenAddress := fmt.Sprintf("%s:%s", config.Get("address"), config.Get("port"))
+	fullListenAddress := fmt.Sprintf("%s:%s", globalConfig.Get("address"), globalConfig.Get("port"))
 
 	http.HandleFunc("/", internal.Landing)
 	http.Handle("/metrics", promhttp.Handler())
 
 	log.Infof("Serving metrics on %s", fullListenAddress)
 	log.Fatal(http.ListenAndServe(fullListenAddress, nil))
 }
+
+func showHelp() {
+	flag.Usage()
+	os.Exit(0)
+}
+
+func showVersion() {
+	if buildDate == "" {
+		buildDate = "at unknown time"
+	}
+	fmt.Printf("version %s\nbuilt with %s %s/%s %s\n", version, runtime.Version(), runtime.GOOS, runtime.GOARCH, buildDate)
```

Comment:
fmt.Printf("version %s\nbuilt with %s %s/%s %s \n", version, runtime.Version(), runtime.GOOS, runtime.GOARCH, buildDate)

Reply:
hmm, I don't think so, why should it? It would leave an invisible empty space at the end of the first line.

Reply:
ok true I think i have misread yesterday